packages/components/src/raw-icon/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+import { cloneElement, createElement, Component, isValidElement } from '@wordpress/element';
+import { Dashicon, SVG } from '../';
+
+function RawIcon( { icon = null, size = 24, className } ) {
```

Comment:
I  suspect the default size property here might have some unexpected side effects on the different types of block icons and the different places where we render them. Mind a sanity check here @jasmussen or @chrisvanpatten as you know more about all the specificities.

Reply:
Interesting timing, I'm actually trying to fix a regression in https://github.com/WordPress/gutenberg/pull/10938#issuecomment-432220310 related to this. 

24 _should_ be default, but it's okay if an SVG needs to override that and show a 20x20 icon. 

If 24 is not default, then that's a bug, as is discussed in that link.

Reply:
It looks like we were explicitly showing Dashicons as `20x20` while this PR makes them `24x24` ?